features/mbedtls/targets/TARGET_STM/TARGET_STM32L4/aes_alt.h

```diff
@@ -0,0 +1,307 @@
+/*
+ *  aes_alt.h AES block cipher
```

Comment:
This file is identical to `features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/aes_alt.h`. Can you think of a way to use a single file for both targets? That'd be far more maintainable. Thanks.